### PR TITLE
[TDV-258] Add blob types constant

### PR DESCRIPTION
**Pull request by ardier16**

### CODE REVIEW

File: src/const/blob-types.const.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+export const BLOB_TYPES = Object.freeze({
+  assetDescription: 'asset_description',
+  fundOverview: 'fund_overview',
+  fundUpdate: 'fund_update',
+  navUpdate: 'nav_update',
```
Comment (quinn71) [inline src/const/blob-types.const.js]: we don't need `navUpdate` and `fundUpdate` types

File: src/const/blob-types.const.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+  fundDocument: 'fund_document',
+  kycSyndicate: 'alpha',
```
Comment (quinn71) [inline src/const/blob-types.const.js]: use "corporate" instead of "syndicate"

File: src/const/blob-types.const.js (hunk 1 above)
Comment (ardier16) [reply src/const/blob-types.const.js]: Removed

File: src/const/blob-types.const.js (hunk 2 above)
Comment (ardier16) [reply src/const/blob-types.const.js]: Renamed

File: src/const/blob-types.const.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
+export const BLOB_TYPES = Object.freeze({
+  assetDescription: 'asset_description',
+  fundOverview: 'fund_overview',
+  fundDocument: 'fund_document',
```
Comment (max-nes) [inline src/const/blob-types.const.js]: `saleDocument`

File: src/const/blob-types.const.js [hunk 4]
```diff
@@ -0,0 +1,8 @@
+export const BLOB_TYPES = Object.freeze({
+  assetDescription: 'asset_description',
+  fundOverview: 'fund_overview',
```
Comment (max-nes) [inline src/const/blob-types.const.js]: `saleOverview`

File: CHANGELOG.md [hunk 5]
```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 Please check our [developers guide](https://gitlab.com/tokend/developers-guide)
 for further information about branching and tagging conventions.
 
+## [Unreleased]
+### Added
+- blob types constant
```
Comment (max-nes) [inline CHANGELOG.md]: `BLOB_TYPES` constant

File: src/const/blob-types.const.js (hunk 4 above)
Comment (ardier16) [reply src/const/blob-types.const.js]: Renamed

File: CHANGELOG.md (hunk 5 above)
Comment (ardier16) [reply CHANGELOG.md]: Renamed

File: src/const/blob-types.const.js (hunk 3 above)
Comment (ardier16) [reply src/const/blob-types.const.js]: Renamed

**Pull request merged**